### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by matthiasbeyer**

### CODE REVIEW

File: lib/talk_helpers.rb [hunk 1]
```diff
@@ -6,3 +6,11 @@ def past_and_next_talks items
     i.attributes[:kind] == :talk and i.attributes[:date] <= limit
   end.sort_by { |i| i.attributes[:date] }
 end
+
+def upcoming_talks items
+  limit = Date.today
```
Comment (TheNeikos) [inline lib/talk_helpers.rb:11]: How is this supposed to work? Update every week?

Comment (matthiasbeyer) [inline lib/talk_helpers.rb:11]: Have a closer look at the diff. This is a filter for getting the _upcoming_ talks. So all talks which are in the future. The method does exactly this!

Comment (TheNeikos) [inline lib/talk_helpers.rb:11]: Sure, but you called the link 'schedule' which is not what this is then.

Comment (matthiasbeyer) [inline lib/talk_helpers.rb:11]: Why not? Schedule translates to "Terminplan" in german, and "upcoming" can be translated as "Noch ausstehend" ... so where's the problem?

Comment (TheNeikos) [inline lib/talk_helpers.rb:11]: That it becomes stale the day after a talk is being held.

Comment (matthiasbeyer) [inline lib/talk_helpers.rb:11]: So you propose that I call the link "upcoming" rather than "schedule"? If yes, please provide a patch to this PR!

Comment (TheNeikos) [inline lib/talk_helpers.rb:11]: Yup! Upcoming makes more sense.

**Comment by manuelmessner** [comment]
The method `past_and_next_talks` has to be changed, otherwise there will be dublicates at the edge cases.
cc @matthiasbeyer

**Comment by matthiasbeyer** [comment]
@TheNeikos @manuelmessner please review my latest changeset.

File: content/talks.html [hunk 2]
```diff
@@ -1,11 +1,24 @@
 ---
-title: Talks
+title: "Historie + Schedule"
```
Comment (TheNeikos) [inline content/talks.html]: Please change the title here to reflect the content

Comment (TheNeikos) [inline content/talks.html]: As in, also call it 'Upcoming'

Comment (matthiasbeyer) [inline content/talks.html]: Fixed. Ready for squashing?

**Comment by matthiasbeyer** [comment]
Rebased onto latest master and cleaned history.

**Pull request merged**